cli/lib/gather.js

```diff
@@ -297,14 +300,23 @@ class Categorizer {
 					// if we are using app thinning, then don't copy the image, instead mark the
 					// image to be injected into the asset catalog. Also, exclude images that are
 					// managed by their bundles.
+
 					if (this.useAppThinning && !relPath.match(BUNDLE_FILE_REGEXP)) {
-						results.imageAssets.set(relPath, info);
-						return;
+						if (this.excludeAssestsDir !== null) {
+							const checkRegEx = new RegExp(this.excludeAssestsDir);
```

Comment:
Can you fix the regex warning?

Reply:
sure... on it...

Reply:
no regex warning....
also added missing part for "png"
Usage in tiapp.xml (no "/" at the start!!!):

```
<ios>
        <exclude-dir-from-asset-catalog>maps/tiles|maps/otherimages</exclude-dir-from-asset-catalog>
```